src/Scope.ts

```diff
@@ -73,6 +72,24 @@ export class Scope {
         return this.getClassFileLink(className, containingNamespace)?.item;
     }
 
+    /**
+     * Get the interface with the specified name.
+     * @param ifaceName - The interface name, including the namespace of the interface if possible
+     * @param containingNamespace - The namespace used to resolve relative interface names. (i.e. the namespace around the current statement trying to find a interface)
+     */
+    public getInterface(ifaceName: string, containingNamespace?: string): InterfaceStatement {
+        return this.getInterfaceFileLink(ifaceName, containingNamespace)?.item;
+    }
+
+    /**
+     * Get either the class or interface with a given name
+     * @param name - The name, including the namespace of the interface if possible
+     * @param containingNamespace - The namespace used to resolve relative names. (i.e. the namespace around the current statement trying to find the interface or class)
+     */
+    public getClassOrInterface(name: string, containingNamespace?: string): InheritableStatement {
```

Comment:
The whole point of this function to get a NAMED type... it will be used for Enums, too, but probably not for anonymousTypes...

Reply:
So once you get enums, would you name this `getClassOrInterfaceOrEnum`? Then, we might eventually support something like typescript:
```typescript
type something = string | integer | float;
```

Anyway, it's v1 so we could rename again in the future if need be, but if you could think of a more generic name, that might be useful for future proofing and for plugins to have one less thing to refactor.

Reply:
I'm renaming it `getNamedTypeStatment`

Reply:
Because this is a lookup for a certain Name, eg... "MyKlass" .. or "MyIFace"  or "MyEnum"... the other types (e.g. `{name as string}` or union types `string | integer` would have no name for lookup, and would be decipherable as a `TypeExpression` (in the future)

Reply:
But imagine you could define them at the top level. That wouldn't be exclusively an expression? Like this:

```brighterscript
interface Person
    name as string
end interface

interface Animal
    name as string
end interface

type Nameable = Person | Animal
```